assets/js/modules/analytics/components/setup/SetupMain.js

```diff
@@ -61,7 +61,13 @@ export default function SetupMain( { finishSetup } ) {
 		select( MODULES_ANALYTICS ).getSetupFlowMode()
 	);
 
-	const { setAccountID } = useDispatch( MODULES_ANALYTICS );
+	const { setAccountID } = useDispatch( MODULES_ANALYTICS_4 );
+	// Temporary added so settings and module setup can work in the meantime before
+	// #7932 is merged. The check access is still connected to analytics module and will
+	// fail to display property and webdatastream options otherwise.
+	const { setAccountID: setAnalyticsAccountID } =
+		useDispatch( MODULES_ANALYTICS );
+
```

Comment:
Same as above, will be removed in #7932, but needed until then

Reply:
We need to keep this fallback temporary as well, like what we are doing in `SelectAccount`